### PR TITLE
Adapt Rafter to 'allowPrivilegedContainers: false' policy

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- as securityContext everywhere
- add psp's+roles+rolebindings with fairly low permissions

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/9552

### CODE REVIEW

File: resources/rafter/charts/controller-manager/charts/minio/templates/role.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "minio.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (aerfio) [inline resources/rafter/charts/controller-manager/charts/minio/templates/role.yaml:5]: add labels

File: resources/rafter/charts/controller-manager/charts/minio/templates/rolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "minio.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (aerfio) [inline resources/rafter/charts/controller-manager/charts/minio/templates/rolebinding.yaml:5]: add labels

File: resources/rafter/charts/controller-manager/charts/minio/templates/psp.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  readOnlyRootFilesystem: false
```
Comment (aerfio) [inline resources/rafter/charts/controller-manager/charts/minio/templates/psp.yaml:32]: flip it

File: resources/rafter/charts/controller-manager/templates/psp.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  readOnlyRootFilesystem: false
```
Comment (aerfio) [inline resources/rafter/charts/controller-manager/templates/psp.yaml:32]: flip

File: resources/rafter/templates/psp.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  readOnlyRootFilesystem: false
```
Comment (aerfio) [inline resources/rafter/templates/psp.yaml:32]: flip

File: resources/rafter/charts/controller-manager/charts/minio/templates/role.yaml (hunk 1 above)
Comment (aerfio) [reply resources/rafter/charts/controller-manager/charts/minio/templates/role.yaml:5]: done

**Review by pPrecel — APPROVED** [review]

**Pull request merged**